presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergConfig.java

```diff
@@ -196,4 +198,17 @@ public HiveStatisticsMergeStrategy getHiveStatisticsMergeStrategy()
     {
         return hiveStatisticsMergeStrategy;
     }
+
+    @Config("iceberg.commit-retries")
+    @ConfigDescription("determines the number of attempts in case of concurrent upserts")
+    public IcebergConfig setConcurrentCommitRetries(int concurrentRetries)
+    {
+        this.concurrentCommitRetries = concurrentRetries;
+        return this;
+    }
+
+    public int getConcurrentCommitRetries()
+    {
+        return concurrentCommitRetries;
+    }
 }
```

Comment:
Since this is a table property for Iceberg to control the retry limit, would it be better to expose this as a table property instead similar to existing Iceberg table properties, WDYT? - https://github.com/prestodb/presto/blob/master/presto-docs/src/main/sphinx/connector/iceberg.rst#table-properties

Reply:
makes sense. Will take care.